### ISSUE TITLE
Stop Displaying Contact's Phone Number.

### ISSUE THREAD

**Issue by iknownothingaboutcomputers** [signalapp/Signal-Android#8014]
Tap on a contact's name and you'll see their phone number displayed below their name on the topmost bar in the conversation page. How is this good for privacy? Makes it hard to even shitpost on Signal in public places.

**Android** : 7.1
**Signal Version**: 4.23.4

**Comment by n-o-o-b** [comment]
Agreed. This shouldn't be displayed like that. We can tap on their name in the conversation if we want to see their number.

**Comment by n-o-o-b** [comment]
But why display it in the conversation if it can be avoided? And arguably the number is more sensitive than the content of the conversation, though obviously this depends. WhatsApp doesn't display the number unelss you tap on the name, and I think this is how it should be.

Also, displaying the number means you can't take a screenshot of a conversation without revealing the person's number, which isn't good.
